modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/IgniteToKafkaCdcStreamer.java

```diff
@@ -84,22 +98,28 @@
     private KafkaProducer<Integer, byte[]> producer;
 
     /** Handle only primary entry flag. */
-    private final boolean onlyPrimary;
+    private boolean onlyPrimary = DFLT_IS_ONLY_PRIMARY;
 
     /** Topic name. */
-    private final String topic;
+    private String topic = DFLT_TOPIC;
 
     /** Kafka topic partitions count. */
-    private final int kafkaParts;
+    private int kafkaParts = DFLT_PARTS;
 
     /** Kafka properties. */
-    private final Properties kafkaProps;
+    private Properties kafkaProps;
 
     /** Cache IDs. */
-    private final Set<Integer> cachesIds;
+    private Set<Integer> cachesIds;
+
+    /** Cache names. */
+    private Collection<String> cacheNames = DFLT_CACHES;
 
     /** Max batch size. */
-    private final int maxBatchSize;
+    private int maxBatchSize = DFLT_MAX_BATCH_SIZE;
+
+    /** The maximum time to complete Kafka related requests, in seconds. */
+    private int kafkaReqTimeout = DFLT_REQ_TIMEOUT;
```

Comment:
Let's measure this timeout in milliseconds

Reply:
Done.